packages/components/modal/src/Modal.styles.ts

```diff
@@ -98,10 +112,10 @@ export function getModalStyles(props: {
         props.overlayClassName,
       ),
       afterOpen: css({
-        opacity: 1,
+        opacity: '1 !important',
```

Comment:
What's the reason for important here? is it being overridden by something?

Reply:
because of the earlier `padding` reset, emotion generates a new class for the fullscreen modal, and it renders below `afterOpen` class, that's why it has bigger specificity. 

<img width="262" alt="CleanShot 2023-05-09 at 13 02 44@2x" src="https://github.com/contentful/forma-36/assets/10744462/44afc107-7f88-4790-928e-51157710b927">

I'm wondering if we can somehow reset the padding for the `fullscreen` differently, so we don't need to hack the specificity, wdyt? 🤔

And btw in storybook, it works fine without `!important` 🤷‍♂️